util/src/entities/User.ts

```diff
@@ -170,11 +170,14 @@ export class User extends BaseClass {
 
 	@Column({ type: "simple-json", select: false })
 	settings: UserSettings;
-		
+
 	// workaround to prevent fossord-unaware clients from deleting settings not used by them
 	@Column({ type: "simple-json", select: false })
 	extended_settings: string;
 
+	@Column({ type: "simple-json" })
+	notes: { [key: string]: string }; //key is ID of user
```

Comment:
You undid the PR to move notes to their own table? Or is this branch not caught up with master?

Reply:
I have no idea. Notes entity seems to exist, might be a merge error.